xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -2156,6 +2156,7 @@ void CVideoPlayer::CheckContinuity(CCurrentStream& current, DemuxPacket* pPacket
        fabs(this_dts - that_dts) < DVD_MSEC_TO_TIME(1000))
     {
       m_offset_pts += correction;
+      m_VideoPlayerVideo->SetOffsetPts(m_offset_pts);
```

Comment:
in https://github.com/Voyager1/xbmc/blob/invisible_dvd_overlays/xbmc/cores/VideoPlayer/VideoPlayerVideo.cpp#L817 : iPTSStartTime and iPTSStopTime are always UNcorrected. The pts we're getting as input parameter is corrected (I discovered it because it was negative after a discontinuity) while start/stop are both >=0...